hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/NodeStatusUpdaterImpl.java

```diff
@@ -399,7 +397,7 @@ nodeManagerVersionId, containerReports, getRunningApplications(),
               nodeLabels, physicalResource, nodeAttributes, nodeStatus);
 
       if (containerReports != null && !containerReports.isEmpty()) {
-        LOG.info("Registering with RM using containers :" + containerReports);
```

Comment:
This is enough to print directly. 

Line#394: getNMContainerStatuses() 
The list of Containers has been printed, no need to reprint.

Line#402: I modified it to print size directly.